packages/db-authorization/index.d.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  type PlatformaticContext,
+  type WhereCondition
+} from '@platformatic/sql-mapper'
+import { type PlatformaticApp } from '@platformatic/types'
+import { type FastifyPluginAsync } from 'fastify'
+import { type FastifyUserPluginOptions } from 'fastify-user'
```

Comment:
This depends on platformatic/fastify-user#5 to land and be bumped first (thus the `wip` for this PR)